app/views/shared/buttons/_ab_form.html.haml

```diff
@@ -1,6 +1,6 @@
 #ab_form
   #policy_bar
-    - if @resolve[:uri] && Hash[*@resolve[:target_classes].flatten].invert[@resolve[:new][:target_class]] == @edit[:new][:target_class]
+    - if session[:resolve_object].present?
```

Comment:
@romanblanco this button should only be available when same target class is selected in the tree as in the options that were copied over, you are missing the condition from the check that was removed in this PR `Hash[*@resolve[:target_classes].flatten].invert[@resolve[:new][:target_class]] == @edit[:new][:target_class]`